### PR TITLE
[USH-920] Complete the steps to enable bot automate adding contributors to the platform client (mzima)

### LINKED ISSUE #626: [USH-920] Complete the steps to enable bot automate adding contributors to the platform client (mzima)
#### Description
All you need to work on is step 3 from the [all contributors doc page](https://allcontributors.org/docs/en/bot/installation.). Don't bother about steps 1 and 2 from the all contributors doc page.

#### Expected solution
In summary:
- Add the .all-contributorsrc file to the root of the [platform client (mzima) repo](https://github.com/ushahidi/platform-client-mzima)
- Follow example from the [backend platform API's .all-contributorsrc file](https://github.com/ushahidi/platform/blob/develop/.all-contributorsrc), to add content to the .all-contributorsrc file you created. Note: The content should be modified to suit the platform client (mzima) repo accordingly, and the contributors property should have an empty array like this instead: `"contributors": []`

<sub>[USH-920](https://linear.app/ushahidi/issue/USH-920/complete-the-steps-to-enable-bot-automate-adding-contributors)</sub>

### PR DESCRIPTION
**Pull request by oyenuga17**
resolves #626 
- I added the .all-contributorsrc file to the root of the [platform client (mzima) repo](https://github.com/ushahidi/platform-client-mzima)
- I made sure the  content  suit the platform client (mzima) repo accordingly, and the contributors property should has an empty array like this instead: "contributors": []

### CODE REVIEW

**Comment by Ifycode** [comment]
Hi @oyenuga17, nice work submitting your first PR for Hacktoberfest 👍🏼🎉

**Review by Ifycode — CHANGES_REQUESTED** [review]
I can see that your pull request changes contains changes made to 2 files. In this case, we only need changes to be made in the .all-contributorsrc file @oyenuga17

**Comment by Ifycode** [comment]
Also just copy the issue title and paste it as the pull request title, so that it is easy to track for me.

**Comment by oyenuga17** [comment]
Please review .. I have reverted the changes  in the package-lock.json @Ifycode

File: .all-contributorsrc [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+  ],
+  "imageSize": 50,
+  "commit": false,
+  "contributorsPerLine": 2,
+  "contributors":[],
+  "projectName": "platform-client-mzima",
+  "projectOwner": "ushahidi",
+  "repoType": "github",
+  "repoHost": "https://github.com"
+}
```
Comment (Ifycode) [inline .all-contributorsrc:14]: Neatly done. Approved! 🎉 Feel free to pick another Hacktoberfest issue to work on.

**Review by Ifycode — APPROVED** [review]

**Pull request merged**